Error when running on lasagna concept exercise

If you go to https://exercism.lol/tracks/java/exercises/lasagna/ and submit the example implementation displayed on the bottom right of the screen, you get the following error:

```
Execution failed for task ':lasagna:compileTestJava'. /opt/test-runner/lib/lasagna/src/test/java/LasagnaTest.java:14: error: illegal start of type
("Remove to run test")
^
/opt/test-runner/lib/lasagna/src/test/java/LasagnaTest.java:14: error: ';' expected
("Remove to run test")
 ^
/opt/test-runner/lib/lasagna/src/test/java/LasagnaTest.java:14: error: illegal start of type
("Remove to run test")
                     ^
/opt/test-runner/lib/lasagna/src/test/java/LasagnaTest.java:14: error: 
...
```

I've tested a bit, and removing the @Ignore("Remove to run test") bit from the tests file will "fix" the error.

--- a/bin/run.sh
+++ b/bin/run.sh
@@ -15,7 +15,7 @@ compiler_error_log="compiler.log"
 cp -r $input_folder /opt/test-runner/lib/$problem_slug
 cd /opt/test-runner/lib
 
-find . -mindepth 1 -type f | grep 'Test.java' | xargs -I file sed -i "s/@Ignore/\n/g" file
+find . -mindepth 1 -type f | grep 'Test.java' | xargs -I file sed -i "s/@Ignore(.*)//g;s/@Ignore//g;" file
 cat <<EOF >settings.gradle
 include '${problem_slug}'
 EOF

